test/groovy/NeoDeploymentTest.groovy

```diff
@@ -82,6 +84,16 @@ class NeoDeploymentTest extends BasePipelineTest {
         cpe = loadScript('commonPipelineEnvironment.groovy').commonPipelineEnvironment
 
         cpe.configuration = [steps:[neoDeploy: [host: 'test.deploy.host.com', account: 'trialuser123']]]
+
+        //
+        // needs to be after loading the scripts. Here we have a different behaviour
+        // for usual steps and for steps contained in the shared lib itself.
+        //
+        // toolValidate mocked here since we are not interested in testing
+        // toolValidate here. This is expected to be done in a test class for
+        // toolValidate.
+        //
+        helper.registerAllowedMethod('toolValidate', [Map], {m -> toolValidateCalled = true;})
```

Comment:
typo ;

Reply:
I cannot see a typo, please help me with finding the typo.

Reply:
I mean using ; with Groovy